packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,11 +1,8 @@
-## v1.0.0-preview.4 (Jun 14, 2023)
-## v1.0.0-preview.3 (Jun 13, 2023)
-## v1.0.0-dev (May 29, 2023)
-## v3.0.0-dev (May 12, 2023)
+## v1.0.0 (Jun 15, 2023)
 
-- Add missing cache invalidation for contexts/customers/login/order [#1073](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1073)
-
-- Upgrade React 18, React DOM 18, @types/react@18, @types/react-dom@v18 Testing library 14 [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
+-   First public release of package on npm under `@salesforce/commerce-sdk-react`
+-   Add missing cache invalidation for contexts/customers/login/order [#1073](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1073)
+-   Upgrade React 18, React DOM 18, @types/react@18, @types/react-dom@v18 Testing library 14 [#1166](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1166)
 
 ## v2.7.1 (May 11, 2023)
```

Comment:
The versioning here is a bit confusing. Do we want to remove it? 
I think the plan is to remove https://www.npmjs.com/package/commerce-sdk-react-preview
So, those versions will not exist anywhere not even in commerce-sdk-react-preview.

Reply:
I think the option for more clarity would be 

```
## @salesforce/commerce-sdk-react@v1.0.0 
```